pxr/usdImaging/usdviewq/appController.py

```diff
@@ -1262,8 +1270,22 @@ def _GetFormattedError(reasons=None):
 
         if not stage:
             sys.stderr.write(_GetFormattedError())
+        return stage
+        
+    def _openStage(self, usdFilePath, sessionFilePath,
+                   populationMaskPaths, muteLayersRe):
+
+        if self._mallocTags != 'none':
+            Tf.MallocTag.Initialize()
+
+        if not usdFilePath:
+            stage = self._getEmptyStage()
+        
         else:
-            stage.SetEditTarget(stage.GetSessionLayer())
+            stage = self._getStageForFile(usdFilePath, sessionFilePath,
+                                          populationMaskPaths, muteLayersRe)
+
+        stage.SetEditTarget(stage.GetSessionLayer())
```

Comment:
It looks like saving overrides saves the root layer only. If someone opens an empty stage, makes some edits via the Interpreter, then does "Save Overrides As", nothing will get written out. We should only create a session layer and set it as the edit target when we have a real stage, and avoid creating the session layer for the empty stage.